### ISSUE TITLE
Telegram iOS - Add option to disable Contact Joined Notifications

### ISSUE THREAD

**Issue by avrahamr** [peter-iakovlev/Telegram#43]
It's really annoying to get notifications every time some long forgotten contact in my phone installed Telegram, good for him, whoever he is.

**Comment by avrahamr** [comment]
Small update: One possible hack is preventing Telegram from accessing your contacts, but it's not without side effects, like unidentified messages...

**Comment by Bashar** [comment]
its there in Android, weird no iOS for such simple important feature

**Comment by sweejin** [comment]
I'd also like to flag this issue. There is no option to disable that notification when some one has installed and joined Telegram. It's slightly annoying since it also creates a new chat thread. Does anyone know a way around this?

**Comment by avrahamr** [comment]
I ended up uninstalling Telegram, I "wonder" why WhatsApp is much more popular...

**Comment by ezdub** [comment]
agreed. I methodically disable notifications for apps that cry wolf. or it's all noise.
Friend-join is information, not notification see: Urgent vs Important. 
vs probably not ever urgent nor important but woke me up 

"no no no, it's not actually a telegram message for you, we just came over to say we heard Sally heard the telegraph was invented. When you do get an actual telegram we'll generate the same obnoxiousness so you won't know the difference."
 
A communication device that makes alerts for non-communications is a major roadblock to adoption of a communication device.

**Comment by avrahamr** [comment]
That's funny, I'm subscribed to this issue to see when @peter-iakovlev will ever notice it, but all I get is more people complaining :-)

**Comment by tillcash** [comment]
+1
